app-android-xamarin/Moodis/Moodis/Feature/History/HistoryActivity.cs

```diff
@@ -52,7 +52,7 @@ private void InitAdapter()
             var layoutManager = new LinearLayoutManager(this);
             recyclerView.SetLayoutManager(layoutManager);
 
-            var adapter = new HistoryStatsAdapter(historyViewModel.FetchStats(SignInViewModel.currentUser.Id, DateTime.Now));
+            var adapter = new HistoryStatsAdapter(historyViewModel.FetchStats(GroupActivityModel.GetFriendIds(), DateTime.Now));
```

Comment:
Same here it checks group stats but, can you still check your own stats, as you should be able to.